modules/kmm-adding-the-keys-for-secureboot.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-adding-the-keys-for-secureboot_{context}"]
+= Adding the keys for secureboot
+
+To use KMM Kernel Module Management (KMM) to sign kernel modules a certificate and private key are required.
+For details on how to create these, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_monitoring_and_updating_the_kernel/signing-kernel-modules-for-secure-boot_managing-monitoring-and-updating-the-kernel#generating-a-public-and-private-key-pair_signing-kernel-modules-for-secure-boot[Generating a public and private key pair].
+
+.Procedure
+
+. Create the `my_signing_key_pub.der` file that contains the certificate and the `my_signing_key.priv` file that contains the private key:
```

Comment:
in the new rhel docs these two files are created and referred to as sb_cert.cer and sb_cert.priv  respectively so we should probably change the names to match that.

Reply:
Changed the file names Please verify that it should be `sb_cert.cer` and not `sb_cert.der`.

Reply:
yes, "sb_cert.cer" how its listed in the RHEL docs so this is consistant with them (its a certificate in the der format so its still accurate)